spa-checklist.md

```diff
@@ -30,20 +30,34 @@ This is a checklist for browser-based Web App **without the backend** or with a
 
 ## Versioning
 
- - [ ] My app has a cache-busting implemented 
+ - [ ] My app has a cache-busting implemented (assets, object cache etc.)
+
+## Assets
+
+ - [ ] Have a 404-page
+ - [ ] Have a [maintenance page](https://www.smashingmagazine.com/2009/06/effective-maintenance-pages-examples-and-best-practices/)
+ - [ ] Images support HDPI screens (Retina, etc)
 
 
 ## Testing
-  - [ ] My app does not have memory leaks
-  - [ ] My app is passing performance tests with high grades
+
+ - [ ] My app does not have memory leaks
+ - [ ] My app is passing performance tests with high grades
```

Comment:
Probably makes sense to add link to [RAIL model](https://developers.google.com/web/fundamentals/performance/rail) in that case